docs/source/data_loaders.rst

```diff
@@ -0,0 +1,85 @@
+.. _dataloader-section:
+
+Acquire and Load Data
+========================
+
+UrbanAccess has data downloaders and data loaders that can be used to acquire and load transit and street network datasets:
+
+1. Transit Network: General Transit Feed Specification (GTFS)
+
+* :ref:`Download <gtfs-download>`
+* :ref:`Load <gtfs-loader>`
+
+2. Street Network: OpenStreetMap (OSM)
+
+* :ref:`Download and Load <osm-download-load>`
+
+.. _gtfs-data-object:
+
+The GTFS Data Object
+~~~~~~~~~~~~~~~~~~~~~~
+
+The GTFS data object stores the processed and aggregated GTFS feed data in Pandas dataframes.
+
+.. autodata:: urbanaccess.gtfs.gtfsfeeds_dataframe.gtfsfeeds_df
```

Comment:
For a couple of these objects, where the string representation shows up (e.g. `<urbanaccess.gtfs.gtfsfeeds_dataframe.urbanaccess_gtfs_df object at 0x0000000007A08588>`), we may want to add the [annotation](http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-autoattribute) option to only show the name. 

Also, minor comment on the actual docstring for this object: I would call it something other than a "series" just for disambiguation with a Pandas Series.